modules/4337/package.json

```diff
@@ -50,27 +50,27 @@
     "@account-abstraction/contracts": "^0.7.0",
     "@noble/curves": "^1.4.0",
     "@nomicfoundation/hardhat-ethers": "^3.0.6",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.10",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
```

Comment:
I ran into some bizzare typescript error again and did a dependency bump. It wasn't it in the end, but I decided to keep it - no breaking changes here.